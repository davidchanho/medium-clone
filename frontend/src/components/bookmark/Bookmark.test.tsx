import React from "react";
import renderer from "react-test-renderer";
import Bookmark from ".";
import { testPost } from "../../tests";

describe("Bookmark", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Bookmark post={testPost} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
