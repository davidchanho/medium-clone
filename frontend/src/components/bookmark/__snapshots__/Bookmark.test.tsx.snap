// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bookmark renders correctly 1`] = `
<span
  onClick={[Function]}
>
  <svg
    fill="currentColor"
    height="22"
    stroke="currentColor"
    strokeWidth="0"
    style={
      Object {
        "color": undefined,
      }
    }
    viewBox="0 0 16 16"
    width="22"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      clipRule="evenodd"
      d="M8 12l5 3V3a2 2 0 00-2-2H5a2 2 0 00-2 2v12l5-3zm-4 1.234l4-2.4 4 2.4V3a1 1 0 00-1-1H5a1 1 0 00-1 1v10.234z"
      fillRule="evenodd"
    />
  </svg>
</span>
`;
